dashboard/templates/metric_selections_human_snip.html

```diff
@@ -29,11 +29,12 @@ <h4>
     <div class="btn-group col-md-3" role="group" aria-label="scan tags" id="scantypeselector">Select scan subtype:</br>
         <span class="text-left">
         {% for scantype in study.scantypes %}
-          {% if scantype.name.startswith('DTI') %}
-            <label style="display:none" class="scantype_dti"> {{ scantype.name }}<input type="checkbox" aria-label="{{scantype.name}}" value="{{scantype.name}}" name="scantype" style="margin-top: 5px;"/></label>
-          {% elif scantype.name in ['IMI', 'RST', 'EMP', 'OBS', 'SPRL', 'VN-SPRL', 'MID', 'SID', 'NBK'] %}
-            <label style="display:none" class="scantype_fmri"> {{ scantype.name }}<input type="checkbox" aria-label="{{scantype.name}}" value="{{scantype.name}}" name="scantype" style="margin-top: 5px;"/></label>
+          {% if scantype.tag.startswith('DTI') %}
+            {% set class = 'scantype_dti' %}
+          {% elif scantype.tag in ['IMI', 'RST', 'EMP', 'OBS', 'SPRL', 'VN-SPRL', 'MID', 'SID', 'NBK'] %}
```

Comment:
Yeah definitely! I left it hardcoded for the moment though since I didn't want to touch the metrics part of the dashboard until we knew what we wanted to do with it.